src/components/PingChart.vue

```diff
@@ -177,7 +185,7 @@ export default {
                     )
                 )
                 .map((beat) => {
-                    const x = this.$root.datetime(beat.time);
+                    const x = this.$root.datetime(beat.time, this.use12HourTimeFormat);
```

Comment:
This change breaks the chart rendering on Firefox. This change is also not necessary, since the time value here is only used for calculating the chart position internally in chart.js.